beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  title: string;
+  gameTime: string;
+  phase: string;
+  season: string;
+  year: string;
+  gameType: string;
```

Comment:
What different types of game types are there?Can we make this an enum as well?

Reply:
it wont be a argument anymore since the game type is only going to be classic